src/main/controllers/helpers/JudgmentHelpers.ts

```diff
@@ -0,0 +1,511 @@
+      {
+        key: {
+          text: translations.description,
+          classes: 'govuk-!-font-weight-regular-m',
+        },
+        value: {
+          text: selectedJudgment.value.sendNotificationUploadDocument[0].value.shortDescription,
+        },
+      },
+      {
+        key: {
+          text: translations.document,
+          classes: 'govuk-!-font-weight-regular-m',
+        },
+        value: {
+          html: downloadLink,
+        },
+      }
+    );
+  }
+  judgmentDetails.push(
+    {
+      key: {
+        text: translations.judgmentMadeBy,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedJudgment.value.sendNotificationWhoMadeJudgement,
+      },
+    },
+
+    {
+      key: {
+        text: translations.name,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedJudgment.value.sendNotificationFullName2,
+      },
+    },
+    {
+      key: {
+        text: translations.sentTo,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedJudgment.value.sendNotificationNotify,
+      },
+    }
+  );
+  return judgmentDetails;
+};
+
+export const getDecisionDetails = (
+  userCase: CaseWithId,
+  selectedDecision: TseAdminDecisionItem,
+  selectedApplicationDocDownloadLink: string | void,
+  selectedApplicationResponseDocDownloadLink: string | void,
+  downloadLink: string,
+  translations: AnyRecord
+): { key: unknown; value?: unknown; actions?: unknown }[][] => {
+  const selectedDecisionApplication = getApplicationOfDecision(userCase, selectedDecision);
+  let responseFrom;
+  if (selectedDecisionApplication?.value.respondCollection?.length) {
+    responseFrom =
+      selectedDecisionApplication?.value.respondCollection[0].value.from === Applicant.CLAIMANT
+        ? translations.responseFromRespondent
+        : translations.responseFromClaimant;
+  }
+  const applicationDetails = [];
+  const responseDetails = [];
+  const decisionDetails = [];
+
+  applicationDetails.push(
+    {
+      key: {
+        text: translations.applicant,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecisionApplication?.value.applicant,
+      },
+    },
+    {
+      key: {
+        text: translations.applicationType,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecisionApplication?.value.type,
+      },
+    },
+    {
+      key: {
+        text: translations.applicationDate,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecisionApplication?.value.date,
+      },
+    },
+    {
+      key: {
+        text: translations.legend,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecisionApplication?.value.details,
+      },
+    }
+  );
+  if (selectedDecisionApplication?.value.documentUpload) {
```

Comment:
For new logical sections like this, give it some space.

Reply:
Fixed